### ISSUE TITLE
When I start the script, it puts my mouse at the centre of my screen.

### ISSUE THREAD

**Issue by Stigurna** [Kelkador/Kid-Icarus-AHK-Scripts#4]
Is this the issue mentioned at the start of the main page? Or did I fuck it up really badly.

The area my mouse can move is also much bigger then what the tiny part of the screen is sized. Is this because I can't set my display to Seperate Screens?

I used an older version of this script earlier, and now I'm trying to set it up brand new. I have Citra Nightly 1772.

**Comment by Kelkador** [comment]
Hi! Apologies for the delay on this but I'm working on a video guide and update aimed to be released this week or next. Just posting to confirm I've seen this but asking for a bit of time.

**Comment by Stigurna** [comment]
> Hi! Apologies for the delay on this but I'm working on a video guide and update aimed to be released this week or next. Just posting to confirm I've seen this but asking for a bit of time.

Yeah of course! The fact you're working on this at all is freaking awesome! Take care of yourself and the update and the video comes out when it's ready!

**Comment by Kelkador** [comment]
I have updated the script. Should be working now.

**Comment by Stigurna** [comment]
Im getting an error when trying to run the AutoHotkey.exe. Says script file not found D:/Documents/Autohotkey.ahk

Running the controller.ahk says Error at line 1.
Line Text: Persistent
Error: This line does not contain a recognized action. 

The program will exit

I set my keyboard key as M and the script keys as 8 9 and 0 respectively.

Am I being dumb?

**Comment by Kelkador** [comment]
You need to open the .ahk script with the version of AHK included in the release.zip. I replicated your error by using an older version.

**Comment by Stigurna** [comment]
Where is the release.zip? Or do you mean the V3.zip

**Comment by Kelkador** [comment]
I meant V3.zip, apologies.
